tests/kani/Intrinsics/Assert/uninit_valid_panic.rs

```diff
@@ -5,11 +5,11 @@
 #![feature(core_intrinsics)]
 use std::intrinsics;
 
-// The code below attempts to leave type `bool` uninitialized, causing the
+// The code below attempts to leave type `!` uninitialized, causing the
 // intrinsic `assert_uninit_valid` to generate a panic during compilation.
 #[kani::proof]
 fn main() {
     let _var: () = unsafe {
-        intrinsics::assert_uninit_valid::<bool>();
+        intrinsics::assert_uninit_valid::<!>();
```

Comment:
The problem is that this triggers the same path as `tests/kani/Intrinsics/Assert/inhabited_panic.rs`. Can you check that you get the message below if you replace `bool` with `&i32`?
```
attempted to leave type `&i32` uninitialized, which is invalid
```

Reply:
Yep! Fixed.